keylime.conf

```diff
@@ -124,6 +124,9 @@ tpm_signing_alg = rsassa
 # create a new EK upon startup, and neither will it flush the EK upon exit.
 ek_handle = generate
 
+# The user account to switch to to drop privileges when started as root
+runas = nobody:nobody
```

Comment:
Maybe change this to `run_as` so it's clearer it's 2 words.

Reply:
Fixed.
I noticed that if this field is missing the config.get() will throw an exception. Are we supposed to surround this with try/except considering how an RPM may handle the update of a config file (I think it won't overwrite it)?

Reply:
That's a good question. I don't think Keylime has been consistent in the past when it's added new configuration directives, but I think going forward we need to be better. If we are adding a new directive, unless absolutely necessary it should have a reasonable default and not be required. So I would say if it's now set or empty, just keep running as root.

Reply:
> That's a good question. I don't think Keylime has been consistent in the past when it's added new configuration directives, but I think going forward we need to be better. If we are adding a new directive, unless absolutely necessary it should have a reasonable default and not be required. So I would say if it's now set or empty, just keep running as root.

I now modified config's get() function to take a fallback parameter that I think we should always give with any new option we are introducing. The docs call it fallback parameter. I would prefer default parameter... 

Also just recently again with some webapp changes I think the webapp errorred out if it didn't have an up-to-date keylime.conf file .